tests/test_basic.py

```diff
@@ -3,13 +3,19 @@
 
 import os
 import sys
-from unittest import TestCase
+from unittest import TestCase, skipIf
 
 from nose.plugins import PluginTester
 from nose_randomly import RandomlyPlugin
 
 fixtures = os.path.join(os.path.dirname(__file__), 'fixtures')
 
+try:
```

Comment:
don't do this, you can assume `numpy` is installed since you added it to `requirements.in`